topics/single-cell/tutorials/bulk-music-2-preparescref/workflows/part2_workflow.ga

```diff
@@ -0,0 +1,1060 @@
+                        "tool_version": null,
+                        "type": "tool",
+                        "uuid": "571a650f-5b76-4103-98f1-7a8400f1abbd",
+                        "workflow_outputs": []
+                    },
+                    "10": {
+                        "annotation": "",
+                        "content_id": "toolshed.g2.bx.psu.edu/repos/bgruening/text_processing/tp_sed_tool/1.1.1",
+                        "errors": null,
+                        "id": 10,
+                        "input_connections": {
+                            "infile": {
+                                "id": 9,
+                                "output_name": "table"
+                            }
+                        },
+                        "inputs": [],
+                        "label": "Final Count Matrix with Gene Symbol ID's",
+                        "name": "Text transformation",
+                        "outputs": [
+                            {
+                                "name": "output",
+                                "type": "input"
+                            }
+                        ],
+                        "position": {
+                            "left": 1284.0125732421875,
+                            "top": 386.07501220703125
+                        },
+                        "post_job_actions": {},
+                        "tool_id": "toolshed.g2.bx.psu.edu/repos/bgruening/text_processing/tp_sed_tool/1.1.1",
+                        "tool_shed_repository": {
+                            "changeset_revision": "f46f0e4f75c4",
+                            "name": "text_processing",
+                            "owner": "bgruening",
+                            "tool_shed": "toolshed.g2.bx.psu.edu"
+                        },
+                        "tool_state": "{\"adv_opts\": {\"adv_opts_selector\": \"basic\", \"__current_case__\": 0}, \"code\": \"1 s/^gene//\", \"infile\": {\"__class__\": \"ConnectedValue\"}, \"__page__\": null, \"__rerun_remap_job_id__\": null}",
+                        "tool_version": null,
+                        "type": "tool",
+                        "uuid": "417aedef-9d94-43b1-a3cd-adef81d7132a",
+                        "workflow_outputs": [
+                            {
+                                "label": "Gene Symbol Count Matrix",
+                                "output_name": "output",
+                                "uuid": "3cefa576-db2d-44f6-8cf2-26c846cf5d88"
+                            }
+                        ]
+                    }
+                },
+                "tags": "",
+                "uuid": "1eccbdc6-6b28-4cbf-abbe-59145eeee57e"
+            },
+            "tool_id": 171553,
```

Comment:
The build error now is related to this, doesn't recognize it as a valid tool, something for us to fix, not you @nomadscientist

Reply:
I've pushed a commit that *should* fix this.

Reply:
Success! Now failing for a much more reasonable reason:

```
in `render': Could not find document 'topics/transcriptomics/tutorials/scrna-preprocessing-tenx/tutorial.md' in tag 'link'. (ArgumentError)
```

Reply:
amazing! thanks!